frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -100,20 +138,34 @@ async function deleteUserRoleAssignment(
                     },
                 }"
             >
-                <TabPanel header="Users">
+                <TabPanel
+                    header="Application admins"
+                    v-if="selectedApplicationShortDisplayText === FAM_APPLICATION_NAME"
+                >
+                    <template #header>
+                        <Icon icon="enterprise" :size="IconSize.small" />
+                    </template>
+                    <ApplicationAdminTable
```

Comment:
May I suggest to have the TabPanel as a custom component that receives as props the header, the icon and the table? That way I think it could be reused for as many tabs as needed for this page with minimal code repetition.